CMakeLists.txt

```diff
@@ -1344,7 +1342,6 @@ configure_linker_script(
   ${APP_SMEM_ALIGNED_DEP}
   ${KOBJECT_LINKER_DEP}
   ${CODE_RELOCATION_DEP}
-  zephyr_generated_headers
```

Comment:
So that the former `zephyr_generated_headers` would be taken into account here in another way ?

Reply:
Thanks. No it should be zephyr_interface here I think.

Reply:
Actually, it already is, see line below.

So the change is that zephyr_interface now also includes offsets.h and version.h.
So zephyr_interface now includes everything that was previously in zephyr_generated_headers.